app.yaml.template

```
@@ -26,6 +26,8 @@ handlers:
 
 libraries:
 - name: django
-  version: "1.3"
+  version: "1.11"
 - name: numpy
   version: "1.6.1"
```

```suggestion
  version: "1.18.1"
```
I think numpy should be updated to latest.

---

Beyond the scope, [See this](https://github.com/sympy/sympy_gamma/pull/135#issuecomment-612584314)